integration_test/requirements-dev.txt

```diff
@@ -0,0 +1,12 @@
+grpcio>=1.26.0
+grpcio-tools>=1.26.0
+protobuf==3.19.3
+pydocstyle >= 6.1.1
+mypy >= 0.910
+mypy-protobuf >= 3.0.0
+types-protobuf >= 0.1.14
+grpc-stubs >= 1.24.7
```

Comment:
@int0x27: Do we really need the versions here?
(Interferes with the hvacservice/requirements-dev.txt

Reply:
not familar with this codebase, but it is considered better style to have version in requirments (I think actually the "correct" way to do this these days is using Pipfile and also checking in Pipflile.lock (which really nails down versions to a specific one). but maybe somtehing for optimisation later

Reply:
Those were taken from python sdk template repository, I guess if it works without any versions we are good.
dapr had some fixed protobuf version requirements (but it seems they have upgraded to recent version now)

Reply:
Yes, in the mid term fixed versions would be better. The problem I had/have is, that we have two Python "projects" in this repo, the HVAC service and the integration tests, which both specify requirements. For building the dev-conti I need to install packages for both and the problem is, that pip is complaining that the versions in both files differ (actually a version is given only in one of both).